helix-query.yaml

```diff
@@ -51,3 +51,12 @@ indices:
         select: head > meta[property="article:tag"]
         values: |
           attribute(el, 'content')
+      image:
+        select: head > meta[property="og:image"]
+        value: attribute(el, "content")
+      career-quote:
```

Comment:
what is the need of this ?

Reply:
We also need the image. It's stored with the normal metadata as it's also the main image of the page. 

The default indexing does this already, I think you have to list it explicitly if you have a helix-query.yaml file (at least it didn't work for me when I tried it without).

Reply:
I was talking about career-quote and career-jobtitle

Reply:
They are needed on the career-carousel which is displayed on a number of pages. This is implementing a suggestion from @sdmcraft to put this info in the index so the carousel can obtain its info from there.

Reply:
Got it, Thanks